cita-executor/core/src/libexecutor/block.rs

```diff
@@ -85,14 +85,20 @@ impl ExecutedBlock {
     pub fn create(
         conf: &BlockSysConfig,
         block: OpenBlock,
-        trie_db: Arc<TrieDB<RocksDB>>,
+        trie_db: Arc<CitaTrieDB>,
         state_root: H256,
         last_hashes: Arc<LastHashes>,
         eth_compatibility: bool,
     ) -> Result<Self, Error> {
+        let state = CitaState::from_existing(Arc::<CitaTrieDB>::clone(&trie_db), state_root)
+            .expect("Get state from trie db");
```

Comment:
Maybe `expect` is not a good idea. It'll panic. That's not what we want

Reply:
But, if it cannot get state correctly, the logic can do nothing!